packages/zero-cache/src/db/specs.ts

```diff
@@ -26,6 +26,12 @@ export const columnSpec = v.object({
   pos: v.number(),
   dataType: v.string(),
   pgTypeClass: pgTypeClassSchema.optional(),
+
+  // If the column is an array, this will be the type of the
+  // elements in the array. If the column is not an array,
+  // this will be null.
+  elemPgTypeClass: pgTypeClassSchema.nullable().optional(),
```

Comment:
and we've punted on supporting arrays of arrays, correct?

Reply:
pg does not support array of array. it supports multi dimensional arrays which is slightly different. Those are strange in pg to say the least